tool/tctl/common/plugins_command.go

```diff
@@ -96,11 +171,147 @@ func (p *PluginsCommand) Cleanup(ctx context.Context, clusterAPI *auth.Client) e
 	return nil
 }
 
+type samlConnectorsClient interface {
+	GetSAMLConnector(ctx context.Context, id string, withSecrets bool) (types.SAMLConnector, error)
+}
+
+type pluginsClient interface {
+	CreatePlugin(ctx context.Context, in *pluginsv1.CreatePluginRequest, opts ...grpc.CallOption) (*emptypb.Empty, error)
+}
+
+type installPluginArgs struct {
+	samlConnectors samlConnectorsClient
+	plugins        pluginsClient
+}
+
+func (p *PluginsCommand) InstallOkta(ctx context.Context, args installPluginArgs) error {
+	oktaSettings := p.install.okta
+
+	if oktaSettings.accessListSync {
+		if len(oktaSettings.defaultOwners) == 0 {
+			return trace.BadParameter("AccessList sync requires at least one default owner to be set")
+		}
+	}
+
+	if oktaSettings.scimToken != "" {
+		if len(oktaSettings.defaultOwners) == 0 {
+			return trace.BadParameter("SCIM support requires at least one default owner to be set")
+		}
+
+		if oktaSettings.appID == "" {
+			return trace.BadParameter("SCIM support requires app-id to be set")
+		}
+
+		if oktaSettings.samlConnector == "" {
+			return trace.BadParameter("SCIM support requires saml-connector to be set")
+		}
```

Comment:
Why samlConnector is only validated when SCIM is enabled ? 

Don't we reference to samlConnector in our okta sync flow ? For instance: 
https://github.com/gravitational/teleport.e/blob/5012ca517540f8d1d44bda375fc5de0e8113e04a/lib/okta/synchronize.go#L640

Reply:
Ok, will just make it flat-out required